docs/products/influxdb.rst

```diff
@@ -0,0 +1,26 @@
+Aiven for Aiven for InfluxDB®
+=============================
+
+What is Aiven for Aiven for InfluxDB®

```

Comment:
I suggest we remove these titles when there are so few words to go with them. Alternatively, expand the sections to give the user more information.